project.clj

```diff
@@ -59,6 +59,7 @@
                  [puppetlabs/jruby-utils "0.1.0"]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/trapperkeeper-authorization "0.7.0"]
+                 [puppetlabs/trapperkeeper-scheduler "0.0.1"]
```

Comment:
seems like it didn't get tagged in the git repo somehow, but yeah, it is on clojars.